adal/authentication_context.py

```diff
@@ -85,6 +88,14 @@ def __init__(
             of the proxy. See http://docs.python-requests.org/en/master/user/advanced/#proxies
             for details.
         '''
+        warnings.warn(
+            """ADAL Python library no longer receives any feature update or bugfix.
+Please use the new library, MSAL Python, available here: https://pypi.org/project/msal/
+
+MSAL Python is easier to use, and more secure.
+Start using MSAL Python by choosing one of the samples that suit your need.
+https://msal-python.readthedocs.io/en/latest/#scenarios

```

Comment:
do you have a link to migration docs or a link to the MSAL Python docs?

Reply:
Good point. I'll add [this link](https://learn.microsoft.com/en-us/azure/active-directory/develop/migrate-python-adal-msal) in my next commit.